workflow/overview.mdx

```diff
@@ -0,0 +1,176 @@
+---
+title: "Workflow Builder Overview"
+sidebarTitle: "Overview"
+---
+
+Our browser-based **Workflow Builder (Editor)** makes it simple to create personalized, automated messaging workflows—no coding required.
+
+## Workflow Elements / Building Blocks
+
+All Novu workflows consist of four basic elements:
+
+- **Workflow Identifier** which is used as the trigger id 
+- A **trigger** that starts the workflow
+- **Channel Step** that send notifications to your configured channels
+- **Action Steps** that control the flow of the workflow and produce state for use in templates 
```

Comment:
Not sure I understand this one here, maybe rephrase it? What does "state for use in templates" means?

Reply:
Addressed.